src/App.test.js

```diff
@@ -3,6 +3,8 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/Hex 2 RGB/i);
   expect(linkElement).toBeInTheDocument();
+  expect(linkElement).toHaveClass('App-Title');
```

Comment:
Well done. You need to decide if you are going to capitalize all parts of a className or just some.